manifests/integreatly-cloud-resources/cloud-resources.package.yaml

```diff
@@ -1,5 +1,5 @@
 channels:
-- currentCSV: cloud-resources.v0.27.1
+- currentCSV: cloud-resources.v0.29.0
```

Comment:
Out of interest, is there a reason why this was set to v0.27.1 rather than v0.28.0?

Reply:
May have been missed on the last CRO bump, we only have this here for our own documentation purposes so we know whats in the bundle. We now use the bundle to install CRO